testdata/p4_16_errors_outputs/issue1944.p4-stderr

```diff
@@ -0,0 +1,4 @@
+issue1944.p4(1): [--Werror=overlimit] error: : value too large for type width
```

Comment:
Should this error message somehow convey that this is a compiler limitation and not a P4 language restriction?